### PR TITLE
QUEST_LV_0100.tsv 3842-7905

### PR DESCRIPTION
**Pull request by zaxxer**
capitals and articles, some sentence improvements

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -3849,62 +3849,62 @@ QUEST_LV_0100_20150317_003848	Talk to the Battle Commander.
...
 QUEST_LV_0100_20150317_003859	Defeat the Large Kepa
 QUEST_LV_0100_20150317_003860	Talk to Laimonas
 QUEST_LV_0100_20150317_003861	Let's listen to Laimonas' favor like Titas said.
 QUEST_LV_0100_20150317_003862	Offer worship to the statue
-QUEST_LV_0100_20150317_003863	Try offering worship to Goddess Zemyna Statue. The Statue is located at the end of the road on the right of Laimonas.
-QUEST_LV_0100_20150317_003864	You offered worship like Laimonas said. Tell Laimnoas about it.
-QUEST_LV_0100_20150317_003865	Listen to the favor of Laimonas.
-QUEST_LV_0100_20150317_003866	Defeat Infrorocktor around the marked area
-QUEST_LV_0100_20150317_003867	Do Laimonas a favor and get rid of the Infro Rocktors on the road to Klaipeda.
+QUEST_LV_0100_20150317_003863	Try offering worship to the statue oh Goddess Zemyna. It is located at the end of the road to the east of Laimonas.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: the statue oh Goddess Zemyna -> the Goddess Zemyna Statue.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -3849,62 +3849,62 @@ QUEST_LV_0100_20150317_003848	Talk to the Battle Commander.
...
+QUEST_LV_0100_20150317_003864	You offered worship like Laimonas said. Report back to him.
+QUEST_LV_0100_20150317_003865	Listen to the favor Laimonas is asking for.
+QUEST_LV_0100_20150317_003866	Defeat Infrorocktors around the marked area
+QUEST_LV_0100_20150317_003867	Do Laimonas a favor and get rid of the Infrorocktors on the road to Klaipeda.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: get rid of the -> defeat

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -3849,62 +3849,62 @@ QUEST_LV_0100_20150317_003848	Talk to the Battle Commander.
...
 QUEST_LV_0100_20150317_003869	You seem to have taken care of Laimonas' favor. Tell the Guard leader that you got rid of the monsters on the road to Klaipeda.
-QUEST_LV_0100_20150317_003870	Defeat Infrorocktor
+QUEST_LV_0100_20150317_003870	Defeat Infrorocktors
 QUEST_LV_0100_20150317_003871	Go to Klaipeda
 QUEST_LV_0100_20150317_003872	Let's move on the way to the Klaipeda.
 QUEST_LV_0100_20150317_003873	Get rid of the Rocktortuga that suddenly appeared
-QUEST_LV_0100_20150317_003874	Please be getting rid of lock torque Two moths that appeared suddenly before entering the Klaipeda.
-QUEST_LV_0100_20150317_003875	Report to Guard Captain the defeat of Rocktortuga.
-QUEST_LV_0100_20150317_003876	Tell Battle Commander to return to camp
+QUEST_LV_0100_20150317_003874	Please get rid of the Rocktortuga that suddenly appeard before the entrance to Klaipeda.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: get rid of the -> defeat the

Comment (laurencee) [inline QUEST_LV_0100.tsv]: Also fix the typo: appeard -> appeared

**Comment by zaxxer** [comment]
^it's done :)

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.

**Pull request closed without merging**